roles/openshift_inventory/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+- no_log: True  # keep task descriptions legible
+  block:
+
+  - name: Standardize on latest variable names
+    set_fact:
+      # goal is to deprecate deployment_type in favor of openshift_deployment_type.
+      # both will be accepted for now, but code should refer to the new name.
+      # TODO: once this is well-documented, add deprecation notice if using old name.
+      deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
+      openshift_deployment_type: "{{ openshift_deployment_type | default(deployment_type) | default(None) }}"
+
+  - name: Normalize openshift_release
+    set_fact:
+      # normalize release if provided, e.g. "v3.5" => "3.5"
+      openshift_release: "{{ openshift_release | string | regex_replace('^v', '') }}"
+    when: openshift_release is defined
```

Comment:
I would rephrase one part here slightly, the main use of openshift_release is for the user to specify generically what major release they want to install or upgrade to, without having to know x.y.z.w levels of precision. That was the intent anyhow. It's mainly just a seed value for cluster setup/upgrade, after which we pin the version to the first master and/or what's already installed.